woocommerce-pdf-invoices-packingslips.php

```diff
@@ -188,8 +188,8 @@ public function includes() {
 		$this->settings = include_once( $this->plugin_path() . '/includes/class-wcpdf-settings.php' );
 		$this->documents = include_once( $this->plugin_path() . '/includes/class-wcpdf-documents.php' );
 		$this->main = include_once( $this->plugin_path() . '/includes/class-wcpdf-main.php' );
+		$this->admin = include_once( $this->plugin_path() . '/includes/class-wcpdf-admin.php' );
 		include_once( $this->plugin_path() . '/includes/class-wcpdf-assets.php' );
-		include_once( $this->plugin_path() . '/includes/class-wcpdf-admin.php' );
```

Comment:
You shouldn't change the load order, this can have unexpected results

Reply:
Ah, sorry, didn't thought about that. Fixed here: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/356/commits/ce7f5e73ba279b03f78513d1688eba63298174bc